package.json

```diff
@@ -62,11 +62,11 @@
     "chai": "^3.5.0",
     "clean-css": "^3.4.9",
     "css-loader": "^0.23.1",
-    "css-modules-require-hook": "^2.1.0",
+    "css-modules-require-hook": "^4.0.0",
     "deep-freeze": "0.0.1",
-    "eslint": "^1.10.3",
-    "eslint-config-airbnb": "^4.0.0",
-    "eslint-plugin-react": "^3.16.1",
+    "eslint": "^2.9.0",
+    "eslint-config-airbnb": "^9.0.1",
```

Comment:
@bransbury `eslint-config-airbnb` have 3 peer dependencies. Without those, eslint is failing. I will merge the PR once you add those peer dependencies.